crypto-first-war.md

```diff
@@ -0,0 +1,61 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | Crypto First War |
+| Headline  | Crypto First War examining the historical context, the rise of Cypherpunks, legislative battles, and the lasting impact on global encryption policies. |
+| Cypherpunktimes | cypherpunktimes.com/...link |
+| Publish Date | 00 Feb 2024 |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | post link… |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement - The importance of securing digital communications became increasingly apparent as computers became more popular. This article looks at the first crypto war, examining the historical context, the rise of Cypherpunks, legislative battles, and the lasting impact on global encryption policies.
+
+Pre-discussion, research and questions
+
+
+# Crypto First War
+
+## Introduction:
+
```

Comment:
Rather than having an introduction maybe use your thesis statement here and then weave these points into the main structure of the article.

Or may be use your thesis statement + "Introduction to the first crypto war in the 90s" as your introduction.

Reply:
Good point I'll update this